modules/odc-accessing-developer-perspective.adoc

```diff
@@ -12,6 +12,9 @@ You can access the *Developer* perspective from the web console as follows:
 .Procedure
 
 . Log in to the {product-title} web console using your login credentials. The default view for the {product-title} web console is the *Administrator* perspective.
-. Use the perspective switcher to switch to the *Developer* perspective. The *Topology* view with options to create an application is displayed.
+. Use the perspective switcher to switch to the *Developer* perspective. The *Topology* view with a list of all the projects in your cluster is displayed.
+. Select an existing project from the list or use the *Project* drop-down list to create a new project.
```

Comment:
Shouldn't this new step be after the image showing the selection of the Developer perspective?

Reply:
Absolutely! Fixed.